static/app/gettingStartedDocs/electron/electron.tsx

```diff
@@ -0,0 +1,98 @@
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Layout, LayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/layout';
+import {ModuleProps} from 'sentry/components/onboarding/gettingStartedDoc/sdkDocumentation';
+import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import {getUploadSourceMapsStep} from 'sentry/components/onboarding/gettingStartedDoc/utils';
+import {t, tct} from 'sentry/locale';
+
+export const steps = ({
+  sentryInitContent,
+}: {
+  sentryInitContent?: string;
+} = {}): LayoutProps['steps'] => [
+  {
+    type: StepType.INSTALL,
+    description: t('Add the Sentry Electron SDK package as a dependency:'),
+    configurations: [
+      {
+        language: 'bash',
+        code: `
+# Using yarn
+yarn add @sentry/electron
+
+# Using npm
+npm install --save @sentry/electron
+        `,
+      },
+    ],
+  },
+  {
+    type: StepType.CONFIGURE,
+    description: (
+      <p>
+        {tct(
+          `You need to call [codeInit:Sentry.init] in the [codeMain:main] process and in every [codeRenderer:renderer] process you spawn.
+           For more details about configuring the Electron SDK [docsLink:click here].`,
+          {
+            codeInit: <code />,
+            codeMain: <code />,
+            codeRenderer: <code />,
+            docsLink: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/electron/" />
+            ),
+          }
+        )}
+      </p>
+    ),
+    configurations: [
+      {
+        language: 'javascript',
+        code: `
+        import * as Sentry from "@sentry/electron";
+
+        Sentry.init({
+          ${sentryInitContent}
+        });
+        `,
+      },
+    ],
+  },
+  // TODO: This step isn't in the original onboarding wizard. Can we just reuse the wizard step? (Still waiting on answers from team)
```

Comment:
Still trying to figure out if this step can be reused in Electron. 

Unrelated:
This also has implications for the source maps wizard and the actual docs.

Reply:
Update: We can just reuse the source maps wizard here as electron apps can be built with various tools and bundlers. We might special-case some of them in the wizard but for the UI, this should do.